<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoMenuWnd</class>
 <widget class="QMainWindow" name="NoMenuWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoMenuWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="BeginWgt" name="beginPage"/>
      <widget class="SetRobotWgt" name="setRobotPage"/>
      <widget class="QWidget" name="page_3"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BeginWgt</class>
   <extends>QWidget</extends>
   <header location="global">beginwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SetRobotWgt</class>
   <extends>QWidget</extends>
   <header location="global">setrobotwgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
